### DISCUSSION TITLE
建议博客配置添加对gitea的需求

### DISCUSSION THREAD

**Discussion by oojun** [Qexo/Qexo#457] [Ideas]
服务器自建gitea ，发现gitea actions 可以使用，能否在博客配置-服务商添加一下对gitea支持。期待一下

![屏幕截图 2023-09-22 233607](https://github.com/Qexo/Qexo/assets/76256558/44a00e1a-d4e3-48fd-a8ac-7f9cfdc3bc3c)

**Comment by zwj326** [comment]
+1

**Reply by am-abudu** [reply]
可以添加一下我的QQ2753534031吗，我可能需要一个测试环境来测试Gitea

**Reply by zwj326** [reply]
> 可以添加一下我的QQ2753534031吗，我可能需要一个测试环境来测试Gitea

抱歉现在才看到，可以的

**Reply by am-abudu** [reply]
现在工作已经完成

**Comment by am-abudu** [comment]
抱歉现在才看到。如果不困难的话应该将加入

**Reply by oojun** [reply]
好像没有推送新版本？在后台找不到相关选项。

谢谢！！

**Reply by am-abudu** [reply]
目前还不确定是否存在问题，先发在Dev分支了。你可以一键更新到Dev体验

**Reply by oojun** [reply]
测试可以连接，我先体验一下，有问题再反馈。

祝你新年快乐🎉
